CMakeLists.txt

```diff
@@ -127,7 +138,7 @@ set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${HDF5_VOL_REST_RESOURCES_MOD_DIR}")
 find_package (YAJL REQUIRED)
 if (YAJL_FOUND)
   include_directories(${YAJL_INCLUDE_DIRS})
-  set (LINK_LIBS ${LINK_LIBS} ${YAJL_LIBRARIES})
+  set (LINK_LIBS ${LINK_LIBS} yajl)
```

Comment:
Should this be `${YAJL_LIBRARIES}` instead of `yajl` to match how curl is linked?